### ISSUE TITLE
Feature request.

### ISSUE THREAD

**Issue by eyvindbjartiandersen** [akrennmair/newsbeuter#514]
Search in tags. I've got alot of tags to navigate and this isn't the fastest way. Fuzzy matching would be preferred.

**Comment by Minoru** [comment]
Yep, that'd be a good feature to have. Thanks for filing the issue about it!
